wagtail/contrib/frontend_cache/tests.py

```diff
@@ -198,6 +198,14 @@ def purge(self, url):
         PURGED_URLS.append(url)
 
 
+class MockCloudflareBackend(CloudflareBackend):
+    def __init__(self, config):
+        pass
+
+    def _purge_urls(self, urls):
+        PURGED_URLS.extend(urls)
```

Comment:
Probably worth raising an exception here if `len(urls) > 30` so that the test would fail if there was no chunking

Reply:
Have added this, thanks!